source/server-selection/server-selection.rst

```diff
@@ -579,9 +579,18 @@ Passing read preference to mongos and load balancers
 ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
 
 If a server of type Mongos or LoadBalancer is selected for a read operation, the read
-preference is passed to the selected mongos through the use of the
-``SecondaryOk`` wire protocol flag, the ``$readPreference`` query
-modifier or both, according to the following rules.
+preference is passed to the selected mongos through the use of ``$readPreference``
+(as a `Global Command Argument`_ for OP_MSG or a query modifier for OP_QUERY) and, for
+OP_QUERY only, the ``SecondaryOk`` wire protocol flag, according to the following rules.
+
+For OP_MSG:
+
+  - For mode 'primary', drivers MUST NOT set ``$readPreference``
```

Comment:
The rendered version of this has these bullets inside of a blockquote block (ditto for the OP_QUERY part). I think they need one more space of indentation or two fewer, I can't really remember with RST.

Reply:
Fixed.